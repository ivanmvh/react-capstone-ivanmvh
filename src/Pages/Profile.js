const Profile = () => (
  <h1>This is the profile</h1>
);

export default Profile;
