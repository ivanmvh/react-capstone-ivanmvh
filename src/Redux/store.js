import { combineReducers, configureStore } from '@reduxjs/toolkit';
import photosSlice from './PhotosSlice';

const photosReducers = combineReducers({
  Photos: photosSlice,
});

const store = configureStore({
  reducer: photosReducers,
});

export default store;
