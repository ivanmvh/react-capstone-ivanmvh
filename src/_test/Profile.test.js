import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import Profile from '../Pages/Profile';
import '@testing-library/jest-dom';
import store from '../Redux/store';

describe('Profile', () => {
  const tree = test('renders Profile component', () => {
    render(
      <Provider store={store}>
        <BrowserRouter>
          <Profile />
        </BrowserRouter>
      </Provider>,
    );
    expect(tree).toMatchSnapshot();
  });
});
