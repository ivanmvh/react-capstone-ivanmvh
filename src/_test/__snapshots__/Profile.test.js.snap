// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders About component 1`] = `undefined`;

exports[`Profile renders Profile component 1`] = `undefined`;
